crosswalks/MODS.csv

```diff
@@ -0,0 +1,68 @@
+﻿Property,MODS
+codeRepository,relatedItem/[@type='Host']

```

Comment:
`codeRepository` should be something like a Git repo, and [this](https://www.loc.gov/standards/mods/userguide/relateditem.html#examples) doesn't seem to match at all.

Reply:
```suggestion
codeRepository,location/url[@access='object in context']
```
hoping this is a more accurate mapping using the [location element and url subelements](https://www.loc.gov/standards/mods/userguide/location.html#locationurl).